sip-core/src/main/java/de/ikor/sip/foundation/core/actuator/routes/AdapterRouteEndpoint.java

```diff
@@ -98,6 +101,11 @@ public void startAll() {
   @GetMapping("/{routeId}")
   @Operation(summary = "Get route details", description = "Get route details")
   public AdapterRouteDetails route(@RouteIdParameter @PathVariable String routeId) {
+    ManagedRouteMBean routeMBean = mbeanContext.getManagedRoute(routeId);
+    if (routeMBean == null) {
+      throw new ResponseStatusException(
+          HttpStatus.NOT_FOUND, "Route with ID:" + routeId + " not found");
```

Comment:
Let's use translations for messages. Especially it they are returned trough API

Reply:
Done

Reply:
Done